src/components/GovernanceOverview/index.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  ProcessList,
+  ProcessListHeading,
+  ProcessListItem
+} from '@trussworks/react-uswds';
+import classNames from 'classnames';
+
+import UswdsReactLink from 'components/LinkWrapper';
+import CollapsableLink from 'components/shared/CollapsableLink';
+import Divider from 'components/shared/Divider';
+
+type processStepProps = {
+  heading: string;
+  content: string;
+  link?: { text: string; path: string };
+};
+
+const GovProcessCollapse = ({ className }: { className?: string }) => {
+  const { t } = useTranslation('governanceOverview');
+  return (
+    <CollapsableLink
+      className={className}
+      id="GovernanceOverview-WhyGovernanceExists"
+      label={t('governanceOverview:processExists:heading')}
+    >
+      <>
+        {t('processExists.subheading')}
+        <ul className="margin-bottom-0 margin-top-1 padding-left-205 line-height-body-5">
+          {(t('processExists.listItems', {
+            returnObjects: true
+          }) as string[]).map(item => {
+            return <li key={item}>{item}</li>;
+          })}
+        </ul>
+      </>
+    </CollapsableLink>
+  );
+};
+
+const GovernanceOverviewContent = ({
+  helpArticle,
```

Comment:
This bool is used for a similar condition in the PrepareForGRT component of a similar type but named `helpMode`. Should we make them all `helpArticle` and use some type enforcement?

Reply:
Good catch! I just pushed changes to update this. I also added a className prop to your PrepareForGRT component for consistency so that we could use the same types.

Reply:
Looks good thank you